RomanASP.py

```diff
@@ -279,7 +265,8 @@ def main():
             snx, sny = cutout_wcs_list[0].toImage(snra, sndec, units='deg')
             pointing, SCA = exposures['Pointing'][0], exposures['SCA'][0]
             array = construct_psf_source(x, y, pointing, SCA, stampsize=size,
-                                         x_center=snx, y_center=sny, sed=sed)
+                                         x_center=snx, y_center=sny,
+                                         sed=sedlist[0])
```

Comment:
Maybe add a comment to this section here that calculates the confusion metric that explains why (and that) you only use the first entry.